packages/type-safe-api/scripts/type-safe-api/generators/generate-next.ts

```diff
@@ -178,6 +178,40 @@ const toPythonName = (namedEntity: 'model' | 'property' | 'operation', name: str
   return nameSnakeCase;
 };
 
```

Comment:
is the list below complete against https://en.wikipedia.org/wiki/List_of_Java_keywords ?

Reply:
It's the same as OpenAPI generator's here for parity :) https://github.com/OpenAPITools/openapi-generator/blob/8f2676c5c2bcbcc41942307e5c8648cee38bcc44/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java#L179

I'll add a comment to the code to link it!